cfgov/cfgov/urls.py

```diff
@@ -397,6 +397,11 @@ def flagged_wagtail_template_view(flag_name, template_name):
     #         include_if_app_enabled('teachers_digital_platform',
     #                                 'teachers_digital_platform.search-urls')),
 
+    flagged_url('TDP_BB_TOOL',
+            r'^practitioner-resources/youth-financial-education/learn-about-the-building-blocks/take-a-tour',  # noqa: E501
+            include_if_app_enabled('teachers_digital_platform',
+                                    'teachers_digital_platform.bb-urls')),
```

Comment:
I think this will cause an error if `bb-urls.py` doesn't exist. I will confirm locally.

Reply:
There’s currently no bb-urls.py in the current 1.0.6 release of TDP, but there should be one in the next release.

Reply:
[Per PEP8](https://www.python.org/dev/peps/pep-0008/#package-and-module-names), module names should never have hyphens in them. Can you please change this file (and, ideally, the other similarly named files [here](https://github.com/cfpb/teachers-digital-platform/tree/master/teachers_digital_platform)) to use underscores instead?

Reply:
I’ll have to include that change with a bump in the TDP version with those filename changes so that nothing breaks.